src/components/AnchorForCommentsOnly/types.ts

```diff
@@ -18,6 +18,8 @@ type AnchorForCommentsOnlyProps = ChildrenProps & {
 
     /** Press handler for the link, when not passed, default href is used to create a link like behaviour */
     onPress?: () => void;
+
+    containsImageLink?: boolean;
```

Comment:
Please add more description for this props
```suggestion
    isImageHasLink?: boolean;
```